src/LMTR_alg.jl

```diff
@@ -131,16 +141,17 @@ function LMTR(
     ∇φ!(g, d) = begin
       jprod_residual!(nls, xk, d, JdFk)
       JdFk .+= Fk
-      jtprod_residual!(nls, xk, JdFk, g) #profiler
+      jtprod_residual!(nls, xk, JdFk, g)
       g
     end
 
     mk(d) = φ(d) + ψ(d)
 
-    # take first proximal gradient step s1 and see if current xk is nearly stationary
+    # Take first proximal gradient step s1 and see if current xk is nearly stationary.
+    # s1 minimizes φ1(d) + ‖d‖² / 2 / ν + ψ(d).
```

Comment:
s1 minimizes φ1(d) + ψ(d)?

Reply:
PG implicitly adds the diagonal quadratic term.